src/python/tests/test_metric_collection.py

```diff
@@ -0,0 +1,75 @@
+"""
```

Comment:
remove these test files for now. possible route: copy the test code over to the main file and comment it out before check in. delete the test files. then push.